pkg/webhook/virtualmachine/virtualmachine.go

```diff
@@ -135,3 +149,12 @@ func (a *virtualMachineAnnotator) InjectDecoder(d types.Decoder) error {
 	a.decoder = d
 	return nil
 }
+
+func containsString(slice []string, s string) bool {
```

Comment:
you are using this function also in the controller please create the function on a utility package and use it from there